packages/kbn-monaco/src/esql/lib/ast/autocomplete/factories.ts

```diff
@@ -95,13 +95,17 @@ export function getAutocompleteCommandDefinition(
   const commandSignature = getCommandSignature(commandDefinition);
   return {
     label: commandDefinition.name,
-    insertText: commandDefinition.name,
+    insertText: commandDefinition.signature.params.length
+      ? `${commandDefinition.name} $0`
+      : commandDefinition.name,
+    insertTextRules: 4, // monaco.languages.CompletionItemInsertTextRule.InsertAsSnippet,
```

Comment:
What is the meaning of this comment? Can you use this exported constant directly?

Reply:
I could but it has a double side effect:
* increase the bundle size of the initial chunk of 100s kbs
* make it highly coupled with `monaco`.

I'm planning to make all the AST high level functions more independent from the monaco specific types, so right now there's only the bundle size concern.

Reply:
No tree shaking? 👎

Reply:
Monaco has a specific setup that disables treeshaking due to some issues.